src/python/WMCore/BossAir/Plugins/SimpleCondorPlugin.py

```diff
@@ -544,6 +541,15 @@ def getProcAds(self, jobList):
 
             ad['TransferOutput'] = "Report.%i.pkl,wmagentJob.log" % job["retry_count"]
 
+            # Do not define Requirements and X509 ads for Volunteer resources
+            if self.reqStr and "T3_CH_Volunteer" not in job.get('possibleSites'):
+                ad['Requirements'] = classad.ExprTree(self.reqStr)
+
+            ad['x509userproxy'] = classad.ExprTree(self.x509Expr % self.x509userproxy)
+            ad['x509userproxysubject'] = classad.ExprTree(self.x509Expr % self.x509userproxysubject)
```

Comment:
and these 2 last lines somehow are not an expression in the condor classads. I guess they are automatically set from the proxy file or something, so classad expr won't work. Still, this one is probably a not a big deal to have in in the volunteer nodes.